src/util/component-types.ts

```diff
@@ -15,6 +15,8 @@ export interface StandardProps {
 	children?: React.ReactNode;
 	/** Styles that are passed through to native control. */
 	style?: React.CSSProperties;
+
+    callbackId?: string,
```

Comment:
Formatting looks a little weird here. Are you using prettier to format stuff on save? I'm sure there's lots of vscode stuff for that.

Reply:
That does look strange. I was lazy about setting up my IDE and was just running the command, and I probably missed this file. Definitely easier to just do it on save so I'll set that up.